### ISSUE TITLE
unsubscribe topic no work

### ISSUE THREAD

**Issue by liseylee** [firebase/quickstart-unity#161]
on iOS  It work well on add topic, but when I unsubscribe topic at using unsubscribeFromTopic:topic function. It  doesn't work at all. I still can receipt the message about the unsubscribe topic.

`[[FIRMessaging messaging] unsubscribeFromTopic:@"order"];`

**Comment by stewartmiles** [comment]
@alexames any ideas on this?

**Comment by alexames** [comment]
Hi Liseylee, 

I tried subscribing and unsubscribing a topic, and then sending a message to that topic after I had unsubscribed, and my app did not receive the message which is what the expected behavior is. If you're still seeing this error try upgrading to a newer version of the SDK and seeing if that makes a difference, or showing us the code you're using to subscribe and unsubscribe, and the log messages too. The log messages should tell you when the subscribe and unsubscribe calls have gone through.
